AlphaWallet/TokenScriptClient/Coordinators/EventSourceCoordinator.swift

```diff
@@ -29,7 +29,7 @@ class EventSourceCoordinator: EventSourceCoordinatorType {
     private let eventsDataStore: EventsDataStoreProtocol
     private var isFetching = false
     private var rateLimitedUpdater: RateLimiter?
-    private let queue = DispatchQueue(label: "com.eventSourceCoordinator.updateQueue")
+    private let queue = DispatchQueue(label: "com.eventSourceCoordinator.updateQueue", qos: .userInteractive)
```

Comment:
Why can't we use the [system queue](https://developer.apple.com/documentation/dispatch/dispatchqueue/2300077-global)?